### PR TITLE
explanation of proposed CI YAML file

### LINKED ISSUE #56: DOC UPDATE:  Explanation of CI File
To ensure volunteers are clear about code, we need an explainer file for the proposed CI file in #49

### PR DESCRIPTION
**Pull request by ProsperousHeart**
explanation of proposed CI YAML file - discussing with CI YAML creator about a couple of things, but wanted to be sure to put this into a draft PR for others to be able to review

resolves #56

### CODE REVIEW

**Comment by ProsperousHeart** [comment]
need to add in info from [this comment](https://github.com/codefordallas/codefordallas.github.io/pull/49/files/1b042e7ce9ef84adb7137c22a355830fc9397882#r1225451771)

**Review by RobinHerzig — COMMENTED** [review]
Looks good, but there's a small typo on line 7:
> "checking code for style errors or **erros** in general"

File: docs/CI_Explainer.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# About This File
+
+In order to assist our volunteers in understanding [the Continuous Integration file](../.github/workflows/ci.yaml), this README will be providing links and additional insights.
+
+This will not be a complete explanation - you can learn more [here](https://docs.github.com/en/actions/learn-github-actions).
+
+[Continous Integration](https://docs.gitlab.com/ee/ci/introduction/) is what developers call the automatic process of linting (checking code for style errors or erros in general), testing, and other checks BEFORE your branch is merged (integrated). This is done usually through platforms that are running "jobs" for these various tasks.
```
Comment (RobinHerzig) [inline docs/CI_Explainer.md]:
```suggestion
[Continuous Integration](https://docs.gitlab.com/ee/ci/introduction/) is what developers call the automatic process of linting (checking code for style errors or errors in general), testing, and other checks BEFORE your branch is merged (integrated). This is done usually through platforms that are running "jobs" for these various tasks.
```

Comment (ProsperousHeart) [inline docs/CI_Explainer.md]:
```suggestion
[Continous Integration](https://docs.gitlab.com/ee/ci/introduction/) is what developers call the automatic process of linting (checking code for style errors or errors in general), testing, and other checks BEFORE your branch is merged (integrated). This is done usually through platforms that are running "jobs" for these various tasks.
```

**Review by RobinHerzig — APPROVED** [review]

**Pull request merged**